examples/codes/envoy-go-extension/build.sh

```diff
@@ -0,0 +1,16 @@
+#!/bin/bash
+
+set -x
+
+function make_build {
```

Comment:
maybe we should use docker to build?
```
envoy-go-extension point$ ./build.sh
+ make_build
+ mkdir build
+ cp ../../../cmd/moe/main/control.go ../../../cmd/moe/main/mosn.go build
+ cp filter.go build
+ cd build
+ GO111MODULE=on
+ CGO_ENABLED=1
+ GOOS=linux
+ GOARCH=amd64
+ go build --buildmode=c-shared -v -o libmosn.so
runtime/cgo
# runtime/cgo
linux_syscall.c:67:13: error: implicit declaration of function 'setresgid' is invalid in C99 [-Werror,-Wimplicit-function-declaration]
linux_syscall.c:67:13: note: did you mean 'setregid'?
/Library/Developer/CommandLineTools/SDKs/MacOSX.sdk/usr/include/unistd.h:593:6: note: 'setregid' declared here
linux_syscall.c:73:13: error: implicit declaration of function 'setresuid' is invalid in C99 [-Werror,-Wimplicit-function-declaration]
linux_syscall.c:73:13: note: did you mean 'setreuid'?
/Library/Developer/CommandLineTools/SDKs/MacOSX.sdk/usr/include/unistd.h:595:6: note: 'setreuid' declared here
```

Reply:
envoy image `mosnio/envoy-go-extension:latest` only provide linux amd64 version